content/cpp/concepts/math-functions/terms/asinh/asinh.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: 'asinh'
```

Comment:
```suggestion
Title: 'asinh()'
```

Reply:
If the entry is about a method/function, the `Title` should include any punctuation such as parentheses and/or periods. 😄